src/pet.js

```diff
@@ -0,0 +1,83 @@
+const MAXIMUM_FITNESS = 10;
+const minHungerLevel = 0;
```

Comment:
Could a few more constants be added here? You could build on what youve started here and make things much more maintainable by adding values for things like the minimum fitness, max age, max hunger etc and referencing these in your code instead of hard coding the values in multiple places. For example your `isAlive()` function could be then made to read something like
```
  get isAlive() {
    return this.age < MAXIMUM_AGE && this.hunger < MAXIMUM_HUNGER && this.fitness > MINIMUM_FITNESS;
  },
```
It reads practically like a sentence this way, much easier to understand whats actually happening 

Also, always use SCREAMING_SNAKE_CASE for your constants, it really makes them stand out!